### PR TITLE
Update userstats.js

### PR DESCRIPTION
**Pull request by thuyvyng**
cleaned up some of the code, also removed buttons for timefrance and replaced it with dropdown. i think it looks cleaner but let me know what u think. i also moved it to the right so it looks cleaner . let me know what u think

Screenshot 2024-08-28 at 9 35 09 AM
Screenshot 2024-08-28 at 9 35 36 AM
Screenshot 2024-08-28 at 9 35 40 AM

### CODE REVIEW

File: src/components/login.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ function Login() {
   return (
     <div>
       <SpotifyAuth
-        redirectUri="https://unpawsthemusic.vercel.app/redirect"
-        clientID={process.env.REACT_APP_API_KEY}
```
Comment (ruangroc) [inline src/components/login.js:13]: probably meant to revert this?

**Review by ruangroc — COMMENTED** [review]
overall I like that there's more space for the color gradient to fill and that

the tabs at the bottom look a bit weird maybe (stark white background on the selected one, hard to see the blue links on the other ones)

the cat and title look misaligned a bit?

File: src/components/login.js (hunk 1 above)
Comment (ruangroc) [reply src/components/login.js:13]: couldn't check the deployed vercel preview bc it couldn't find localhost

Comment (thuyvyng) [reply src/components/login.js:13]: whoops

**Comment by thuyvyng** [comment]
Screenshot 2024-09-05 at 3 14 42 PM


pretty small changes, i updated it to have drop down for time instead of the buttons

**Comment by thuyvyng** [comment]
Screenshot 2024-09-05 at 4 35 18 PM

**Review by ruangroc — APPROVED** [review]
I like it!

**Pull request merged**